docs/user-guide/cli-installcli.md

```diff
@@ -54,37 +54,15 @@ Use the following procedure to install Zowe CLI from an npm registry:
 
    Zowe CLI is installed.
 
-2. (Optional) Check [npmjs.com](https://www.npmjs.com/) for any Zowe plug-ins that have an `@next` version available. If an `@next` version is available, you can install it:
-
-    ```
-    zowe plugins install @zowe/<plugin-name>@next
-    ```
-
-    If no `@next` version is available,  install the `@latest` version of the plug-in:
-
-    ```
-    zowe plugins install @zowe/<plugin-name>
-    ```
-
-    Optional plug-ins are installed.
-
-3. Migrate your Zowe CLI profiles from your current installation to your @next installation. Issue the following command:
+2. (Optional) Address the [Software requirements for CLI plug-ins](../user-guide/cli-swreqplugins.md). You can install most plug-ins without meeting the requirements. However, the plug-ins will not function until you configure the back-end APIs. The IBM Db2 plug-in requires additional configuration to install.
 
+3. (Optional) To install all available plug-ins to Zowe CLI, issue the following command:
+   
    ```
-   zowe config convert-profiles
+   zowe plugins install @zowe/cics-for-zowe-cli@zowe-v2-lts @zowe/db2-for-zowe-cli@zowe-v2-lts @zowe/ims-for-zowe-cli@zowe-v2-lts @zowe/mq-for-zowe-cli@zowe-v2-lts @zowe/zos-ftp-for-zowe-cli@zowe-v2-lts
    ```
 
-   **Note:** Profile data is backed up in case you want to revert the profiles to your previous Zowe CLI installation.
-
-4. (Optional) If you no longer require the profiles for your previous Zowe CLI installation, you can delete them.
-
-   **Important:** We do not recommend deleting the profiles for your previous Zowe CLI installation until you have tested your @next installation and are satisfied with its performance.
-
-   Issue the following command:
-
-   ```
-   zowe config convert-profiles --delete
```

Comment:
Thanks for moving the mention of `convert-profiles` to the updating CLI doc 🙂 I believe it also needs to be removed from the section below about installing from a TGZ.

Reply:
done